### ISSUE TITLE
[QTDOPQ % Sales] Reference Issue

### ISSUE THREAD

**Issue by samcsheng** [sql-bi/Bravo#540]
### Describe the problem

The auto-generated Time Intelligence Measure `[QTDOPQ % Sales]` reference `[PWC Sales]` as the base for calculating variance versus Previous Quarter Complete Sales, which should be `[PQC Sales]` instead.

``` SQL
QTDOPQ % Sales = 
DIVIDE (
    [QTDOPQ Sales],
    [PWC Sales]
)
```
Also, the equivalent measure `[WTDOPW % Sales]` is missing.

Hopefully this will be fixed in the future release ;)

Thanks a lot for creating such a wonderful tool like Bravo!

Sam

### Bravo version

1.0.0 (1.0.8339.28087)

### Tabular model

Power BI Desktop, import mode

### Power BI

Power BI Desktop version October 2022

### Screenshots

![image](https://user-images.githubusercontent.com/118415964/202351635-d9ce41d9-fc48-4c66-9d8d-c22328f073f1.png)

**Comment by albertospelta** [comment]
Hi @samcsheng, we fixed the `[QTDOPQ %]` and added the missing `[WTDOPW %]` measure.
The fix will be available in next release, thank you!

**Comment by albertospelta** [comment]
Fixed in version [1.0.1](https://github.com/sql-bi/Bravo/releases/tag/v1.0.1)
